napi_from_js.md

```diff
@@ -365,3 +257,77 @@ status = napi_new_instance(env, constructor, argc, argv, &value);
 
 #### Return value
 - `napi_ok` if the API succeeded.
+
+### *napi_make_callback*
+
+#### Signature
+```C
+napi_status napi_make_callback(napi_env env,
+                               napi_value recv,
+                               napi_value func,
+                               int argc,
+                               const napi_value* argv,
+                               napi_value* result)
+```
+
+#### Parameters
+- `[in]  env `: The environment that the API is invoked under
+- `[in]  recv`: The `this` object passed to the called function
+- `[in]  func`: `napi_value` representing the JavaScript function
+to be invoked
+- `[in]  argc`: The count of elements in the `argv` array
+- `[in]  argv`: Array of JavaScript values as `napi_value` 
+representing the arguments to the function. If there are more arguments
+than the provided argument count, then only the requested number of 
+arguments are copied. If there are fewer arguments provided than 
+claimed, the rest of `argv` is filled with `napi_value` values that 
+represent `undefined`
+- `[out] result`: `napi_value` representing the JavaScript object returned
+
+#### Return value
+- `napi_ok` if the API succeeded.
+
+#### Description
+This method allows you to call a JavaScript function object from your native 
+add-on. This is the primary mechanism of calling back *from* the add-on's
+native code *into* JavaScript. It is a fairly simple wrapper around
+`node::MakeCallback`. This API is very similar to `napi_call_function`. However,
+it has additional enhancements, like being aware of domains and AsyncHooks.
+
+A sample use case might look as follows. Consider the following JavaScript
+snippet:
+```js
+function AddTwo(num)
+{
+    return num + 2;
+}
+```
+
+Then, the above function can be invoked from your native add-on using the
+following code:
+```C
+// Get the function named "AddTwo" on the global object
+napi_value global, add_two, arg;
+napi_status status = napi_get_global(env, &global);
+if (status != napi_ok) return;
+
+status = napi_get_named_property(env, global, "AddTwo", &add_two);
+if (status != napi_ok) return;
+
+// var arg = 1337
+status = napi_create_number(env, 1337, &arg);
+if (status != napi_ok) return;
+
+napi_value* argv = &arg;
+size_t argc = 1;
+
+// AddTwo(arg);
+napi_value return_val;
+status = napi_make_callback(env, global, add_two, argc, argv, &return_val);
```

Comment:
Good feedback - I'll remove the example- I think the async helper API example should cover napi_make_callback usage- @mhdawson what do you think?